npm/Fable.Import.Electron.fs

```diff
@@ -516,10 +516,10 @@ module Electron =
         | Normal | Separator | Submenu | Checkbox | Radio
 
     and [<StringEnum>] MenuItemRole =
-        | Undo | Redo | Cut | Copy | Paste | Selectall | Minimize | Close
+        | Undo | Redo | Cut | Copy | Paste | Pasteandmatchstyle | Selectall | Delete | Minimize | Close | Quit | Reload | Forcereload | Toggledevtools | Togglefullscreen | Resetzoom | Zoomin | Zoomout | EditMenu | WindowMenu
```

Comment:
> It would be helpful to have a style guide for things like this

For hand written code, the [F# component design guidelines](http://fsharp.org/specs/component-design-guidelines) are probably the most obvious choice.